### PR TITLE
Get ClassifyAndLocalizeImages.py to run (for me)

### PR DESCRIPTION
**Pull request by kpaschen**
I don't know if you use ModelTester currently, I just wanted something so I could check whether the model I've half-trained now can read my handwriting.

* need to use tensorflow.keras model loading here for models saved using that library
* scipy.misc.imresize has been removed; rather than specify an older version of scipy in requirements.txt, I've updated to imageio.imread. It would probably be better to use the image generator in the same way that TrainModel does
* when I load individual images, I don't always have a bounding box, so I put 'if's around the bits that need a bounding box
* updated the class labels list to be obtained via os.listdir at runtime (same as model trainer). This is brittle because the model under test could have been trained using an older set of class labels; might be worth saving that list along with the model?

### CODE REVIEW

**Comment by apacha** [comment]
Looks good. Thanks for the PR
